app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+import shutil
+
+
+def copy_file(command: str) -> None:
+    if command.startswith("cp "):
+        _, source_file, destination_file = command.split()
```

Comment:
your program still may crash if there is more or less then exactly 3 words in teh command.
THe right design for this would be:
1. Split a command.
2. Check if command has exactly 3 elements, starts with cp and files to copy are not the same
3. rest of code here